frontend/packages/core/package.json

```diff
@@ -38,12 +40,12 @@
     "material-table": "^1.60.0",
     "react": "^16.8.0",
     "react-dom": "^16.8.0",
-    "react-hook-form": "^5.7.2",
-    "react-hook-form-devtools": "^1.2.0",
+    "react-hook-form": "^6.3.0",
     "react-is": "^16.8.0",
     "react-router": "^6.0.0-alpha",
     "react-router-dom": "^6.0.0-alpha",
     "styled-components": "^5.1.1",
+    "superstruct": "~0.8.4",
```

Comment:
we need to include this until https://github.com/react-hook-form/resolvers/issues/20 is resolved